### DISCUSSION TITLE
error Copying and pasting a file

### DISCUSSION THREAD

**Discussion by szmrzsik** [freeplane/freeplane#713] [Help]
Copying and pasting a file does not work correctly with the method described below:
     From File Explorer
         Movement by supported mouse click dragged to the node of a Freeplane map
         Creates a copy of the file in "NameDelaCarte_Files"
             Instead of being limited to a link
     The method that gives the expected result is by
         copy file from file explorer
         then right click on a node and paste
Sorry for the english mistakes

**Comment by dpolivaev** [comment]
I think that different behaviors are intentional to give you a choice.

**Reply by szmrzsik** [reply]
Ok thanks for the info, it would seem better to warn the user
